src/org/usfirst/frc/team4915/stronghold/commands/AutoCommand1.java

```diff
@@ -27,104 +27,43 @@ public AutoCommand1(Autonomous.Type type, Autonomous.Strat strat, Autonomous.Pos
         this.type = type;
         System.out.println("Angle: " + position + "Field Position " + position + "strategy " + strat + "Obstacle " + type);
 
-        if (ModuleManager.INTAKELAUNCHER_MODULE_ON) {
-          //  Robot.intakeLauncher.launcherSetNeutralPosition(); // placeholder
-                                                               // for setting
-                                                               // the launcher
-                                                               // to neutral
-                                                               // driving
-                                                               // position
-        }
+        boolean launcherWantsTravelPosition = getLauncherBeginPosition(type);
+
+        if(launcherWantsTravelPosition)
+            addSequential(new LauncherGoToTravelPositionCommand());
+        else
+            addSequential(new LauncherGoToNeutralPositionCommand());
 
     	switch (strat) {
-		case DRIVE_SHOOT_VISION: // sets us up to use vision to shoot a high
-									// goal
-		    //if it is low bar launcher will go to travelPosition
-		if (getLauncherBeginPosition(type) == true){
-		    addSequential(new LauncherGoToTravelPositionCommand());
-            addParallel(new AimLauncherCommand());
-            addSequential(new WaitCommand(.75));
+        case NONE:
+            break;
+
+        case DRIVE_ACROSS:
+            System.out.println("Starting Move Straight");
```

Comment:
I feel like `Move Straight` is the wrong terminology -- `DRIVE_ACROSS` is what's actually being run, not a `MOVE_STRAIGHT` operation.